Hymns/Display/DisplayHymnView.swift

```diff
@@ -27,9 +35,29 @@ struct DisplayHymnView: View {
                     } else {
                         viewModel.currentTab.content
                     }
+                    if !self.showSoundCloud && self.initiatedSoundCloud && AVAudioSession.sharedInstance().isOtherAudioPlaying {
```

Comment:
from the docs:

```    /* Will be true when another application is playing audio.
    Note: As of iOS 8.0, Apple recommends that most applications use secondaryAudioShouldBeSilencedHint instead of this property.
    The otherAudioPlaying property will be true if any other audio (including audio from an app using AVAudioSessionCategoryAmbient)
    is playing, whereas the secondaryAudioShouldBeSilencedHint property is more restrictive in its consideration of whether 
    primary audio from another application is playing.  
    For additional information, see https://developer.apple.com/library/content/qa/qa1882/_index.html
    */
    @available(iOS 6.0, *)
    open var isOtherAudioPlaying: Bool { get }
```
pls fix.

Reply:
Yeah, I had seen this. The problem is that with a WebView it does not change the bool secondaryAudioShouldBeSilencedHint. So that bool will always be false whether we have music playing or not.   

However, we may want to include some logic with this. To distinguish when outside music is playing and contrast that with isOtherAudioPlaying to discern whether it is our WebView playing the other audio or an outside app

Reply:
> However, we may want to include some logic with this. To distinguish when outside music is playing and contrast that with isOtherAudioPlaying to discern whether it is our WebView playing the other audio or an outside app

How do you think we should distinguish this?